QualityControl/lib/database/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file 'COPYING'.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * Initializes the database connection and runs migrations.
+ * @param {object} sequelizeDatabase - The Sequelize database instance.
+ * @returns {Promise<void>} A promise that resolves when the database is initialized.
+ */
+export const initDatabase = async (sequelizeDatabase) => {
+  try {
+    await sequelizeDatabase.connect();
+    await sequelizeDatabase.migrate();
+  } catch {
+    process.exit(1);
```

Comment:
I do not think this should control the server but instead log the error. 
The QC GUI should still be up and running and the about page should be the one stating that DB connection is down or something bad happened